astropy/coordinates/builtin_frames/intermediate_rotation_transforms.py

```diff
@@ -129,3 +147,26 @@ def precessedgeo_to_gcrs(from_coo, to_frame):
 
     # then move to the GCRS that's actually desired
     return gcrs_coo.transform_to(to_frame)
+
+
+@frame_transform_graph.transform(FunctionTransformWithFiniteDifference, TEME, ITRS)
```

Comment:
Just to be sure: this is a function rather than analytical because of its dependent on obstime, right?

Reply:
Partly, though for reasons I don't fully understand the velocities don't come out right when I use a DynamicMatrixTransform. I have a hunch that the transformation of velocities isn't right for non-intertial frames, but I haven't dug into it enough yet to raise an issue.

Reply:
Maybe raise an issue for now just to remind us...

Reply:
xref: gh-10181